README.md

```diff
@@ -32,16 +32,28 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
-1. What is the difference between an adaptive website and a fully responsive website?
+1. What is the difference between an adaptive website and a fully responsive website? 
+
+An adaptive website is one that can accommodate many different devices, but needs to be formatted for all those devices. They are hard coded in to respond to all kinds of different devices. Alternatively, a responsive website uses a viewport and media queries to adjust to mobile, tablet, desktop versions. You have a much faster and smoother viewing experience. 
```

Comment:
So technically, both adaptive and fully responsive websites use media queries - the difference is what happens between the media breaks. Fully responsive websites are fluid (meaning they use percentages to fit respond to the screen widths) between breakpoints while adaptive are more all-or-nothing.